pages/index.vue

```diff
@@ -1,26 +1,258 @@
+              ability to use boolean operations without ever loosing the
+              original shape.
+            </p>
+          </li>
+          <li>
+            <img src="~/assets/img/icon-guides.png" alt="Guides icon" />
+            <h3>Pixel Precision</h3>
+            <p>
+              An always crip and grid-hinted design thanks to the built-in pixel
+              grid, rulers, and guidelines. Becuase you know, sub-pixels are not
+              a thing.
+            </p>
+          </li>
+          <li>
+            <img src="~/assets/img/icon-branches.png" alt="Git branches icon" />
+            <h3>Built-in Version Control</h3>
+            <p>
+              Never lose your work again with the Akira file format and its
+              built-in local Git repository
+            </p>
+          </li>
+          <li>
+            <img src="~/assets/img/icon-wireframe.png" alt="Wireframe icon" />
+            <h3>Intuitive interface</h3>
+            <p>
+              No need to get lost in hundreds of menus or dropdowns, watch the
+              interface follow your actions and boost your productivity.
+            </p>
+          </li>
+        </ul>
+
+        <a href="features" class="cta">Learn more</a>
+      </section>
+
+      <section class="team">
+        <h2 class="container mx-auto">Team</h2>
+
+        <p>
+          Akira exists thanks to the contribution of hundreds of developers and
+          designers, but the four guys that started all of this need a bit of a
+          shout out.
+        </p>
+
+        <ul>
+          <li>
+            <img
+              src="~/assets/img/avatar-albertoFanjul.png"
+              alt="Alberto Fanjul photo"
+            />
+            <h3>Alberto Funjul</h3>
+          </li>
+          <li>
+            <img
+              src="~/assets/img/avatar-alessandroCastellani.png"
+              alt="Alessandro Castellani photo"
+            />
+            <h3>Alessandro Castellani</h3>
+          </li>
+          <li>
+            <img
+              src="~/assets/img/avatar-bilalElmoussaoui.png"
+              alt="Bilal Elmoussaoui photo"
+            />
+            <h3>Bilal Elmoussaoui</h3>
+          </li>
+          <li>
+            <img
+              src="~/assets/img/avatar-felipeEscoto.png"
+              alt="Felipe Escoto photo"
+            />
+            <h3>Felipe Escoto</h3>
+          </li>
+        </ul>
+
+        <a href="team" class="cta">Contributors</a>
+      </section>
+
+      <section class="support">
+        <h2>Support us</h2>
+
+        <p>
+          Akira is a Free and Open Source Project that aims at offering a
+          professional and native Linux alternative to proprietary UX Design
+          apps. Akira is made possible by the work of talented developers, help
+          us to keep this project alive.
+        </p>
+
+        <a href="https://www.patreon.com/akiraux/posts" class="cta">Patreon</a>
```

Comment:
please add a `target="_blank"` and a descriptive `title` tag to all the external links

Reply:
Ok I've addressed this I think